src/os/ObjectStore.h

```diff
@@ -1358,6 +1363,36 @@ class ObjectStore {
       }
       data.ops++;
     }
+
+   /**
+    * Move temporary object to base object.
+    * Data portion is only copied from temp object to base object.
+    * The copy is done according to the move_info vector of tuple, which
+    * has information of src_offset, dest_offset and length.
+    * Finally, the temporary object is deleted.
+    */
+    void move_tempobjs(const coll_t& temp_cid, const ghobject_t& temp_oid, coll_t cid,
```

Comment:
Again, it may not be a temp object -- temp_cid might be the same as cid.  Better to name them source_cid/source_oid.  Also, why is temp_cid passed as a const & while cid is passed by value?  Pass both by const &.